src/yxy.rs

```diff
@@ -60,7 +60,7 @@ impl<T: Float> Alpha<Yxy<T>, T> {
 
 impl<T: Float> FromColor<T> for Yxy<T> {
     fn from_xyz(xyz: Xyz<T>) -> Self {
-        let mut yxy = Yxy{ luma: xyz.y, ..Default::default() };
+        let mut yxy = Yxy{ x: T::zero(), y: T::zero(), luma: xyz.y };
```

Comment:
Well sort of. I am not sure what the right values here are. If you don't make them 0's there will be a big difference between XYZ(0, 0, 0) and XYZ(0, 0, 0.0001) converted Yxy , that's one of the reasons I made them zero's.